### DISCUSSION TITLE
External non-solaredge power meter

### DISCUSSION THREAD

**Discussion by deanZZZZZ** [WillCodeForCats/solaredge-modbus-multi#480] [General]
I have 1 solaredge inverter and a shelly pro 3em to measure consumption.
Can I use the external Shelly meter for zero export??

**Comment by WillCodeForCats** [comment]
You could probably hack something together with home assistant automatons that compare power values and adjust the inverter output accordingly.

If you mean "configure" as in set the inverter do that automatically, no, the inverter would need a meter it can read directly.

**Reply by WebSpider** [reply]
@WillCodeForCats are you aware of anyone who has used home-assistant automations to compare power values and adjust the inverter output? I'm considering building an automation to limit production when energy prices are negative, which is a possibility in my country. This effectively means I would have to pay to dump my electricity into the grid, and I'd like to prevent that.

My main concern is the update interval to the inverter I guess. I get data from my energy meter every 10 seconds, but I guess it's not a good idea to change the limit of the inverter quite that often?

**Reply by WillCodeForCats** [reply]
There shouldn't be any harm in adjusting the Active Power Limit (`number.solaredge_sim_i1_active_power_limit`) constantly since it's a dynamic setting.
